src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -187,15 +187,30 @@ private void DynamoModel_LanguageDetected()
         }
 
         private void WebView_NavigationCompleted(object sender, CoreWebView2NavigationCompletedEventArgs e)
-        {            
-            if (webView != null)
+        {
+            try
+            {
+                // Exceptions thrown in WebView_NavigationCompleted seem to be silenced somewhere in the webview2 callstack.
+                // If we catch an exceptions here, we log it and close the spash screen.
+                //
+                if (webView != null)
+                {
+                    webView.NavigationCompleted -= WebView_NavigationCompleted;
+                    webView.Focus();
+                    System.Windows.Forms.SendKeys.SendWait("{TAB}");
+                }
+                OnRequestDynamicSplashScreen();
+            }
+            catch (Exception ex)
             {
-                webView.NavigationCompleted -= WebView_NavigationCompleted;
-                webView.Focus();
-                System.Windows.Forms.SendKeys.SendWait("{TAB}");
+                if (!DynamoModel.IsCrashing && !IsClosing)
+                {
+                    CrashReportTool.ShowCrashWindow(viewModel, new CrashErrorReportArgs(ex));
+                    Close();// Close the SpashScreen
```

Comment:
added Try Catch in the `WebView_NavigationCompleted` body.
Looks like all exceptions thrown in WebView_NavigationCompleted are silenced by webview2.

`SplashScreen.WebView_NavigationCompleted` is particularly sensitive because Dynamo views are initialzied inside this method and if an exceptions is thrown, SplashScreen will seem like it hangs.